core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -388,6 +388,16 @@ public void setStateTest() throws Exception {
   }
 
   /**
+   * Tests the permission bits are 0755 for directories and 0644 for files by default.
+   */
+  @Test
+  public void permissionTest() throws Exception {
+    mFileSystemMaster.createFile(NESTED_FILE_URI, sNestedFileOptions);
+    Assert.assertEquals(0755, mFileSystemMaster.getFileInfo(NESTED_URI).getPermission());
```

Comment:
if that's the case, should we be using `PermissionStatus.applyDirUMask(Constant.defaultPermission())` in place of 0755 and `PermissionStatus.applyFileUMask(Constant.defaultPermission())` in place of 0644 everywhere? Except for one place in PermissionStatusTest where we actually check that `PermissionStatus..applyDirUMask(Constant.defaultPermission())` equals 0755 (and similarly for the file umask).